config/bin_version_strings.cfg

```diff
@@ -755,4 +755,5 @@ zipnote;;unknown;"ZipNote\ [0-9](\.[0-9]+)+?\ ";"sed -r 's/ZipNote\ ([0-9](\.[0-
 zipsplit;;bsd-style;"ZipSplit\ [0-9](\.[0-9]+)+?";"sed -r 's/ZipSplit\ ([0-9](\.[0-9]+)+?)/info-zip:zip:\1/'";
 zlib;;zlib;"deflate\ [0-9](\.[0-9]+)+?\ Copyright.*Mark\ Adler";"sed -r 's/deflate\ ([0-9](\.[0-9]+)+?)\ .*/zlib:\1/'";
 zlib;;zlib;"inflate\ [0-9](\.[0-9]+)+?\ Copyright.*Mark Adler";"sed -r 's/inflate\ ([0-9](\.[0-9]+)+?)\ .*/zlib:\1/'";
+zlib;;zlib;"^\ unzip\ [0-9]\.([0-9]+)\ Copyright 1998-20[0-9][0-9] Gilles Vollant.*$";"sed -r 's/unzip\ ([0-9])\.([0-9])([0-9]*)\ Copyright 1998-20[0-9][0-9] Gilles Vollant.*/zlib:\1.\2.\3/'";
```

Comment:
How does a version identifier looks like? From the web page I have found it should be something like 1.2.3 but your grep regex will only identify something like 1.2 or 1.23.
Additionally, I am wondering if we can simplify the sed command in the following way:
`"sed -r 's/unzip\ ([0-9](\.[0-9][0-9]+)+?)\ Copyright 1998-20[0-9][0-9] Gilles Vollant.*/zlib:\1/'"`
Not sure if this fits ...

Reply:
The string that I found was something like "unzip 1.01" and with sed I replace it with zlib 1.0.1